beacon_chain/state/state_transition.py

```diff
@@ -109,7 +109,7 @@ def validate_attestation(crystallized_state: CrystallizedState,
         if has_voted(attestation.attester_bitfield, i)
     ]
     message = blake(
-        in_cycle_slot_height.to_bytes(8, byteorder='big') +
+        attestation.slot.to_bytes(8, byteorder='big') +
```

Comment:
[L105](https://github.com/ethereum/beacon_chain/blob/17369c04158fe7162a449d1e4eee50e4344783d9/beacon_chain/state/state_transition.py#L105): `in_cycle_slot_height` is assigned to but never used.